action.yml

```
@@ -16,6 +16,11 @@ inputs:
     required: false
     type: boolean
     default: false
+  pre-gap:
+    description: 'Commands to be executed immediately before GAP is launched (e.g. ''valgrind --trace-children=yes --leak-check=full'')'
```

Do those double single quite `''` really work?

---

Looking at the code, it seems this description is a bit misleading; this command is not so much executed before GAP, but rather the string is prepended before 'gap' and must act as a "launcher" for GAP.

And also the plural, "commands" -- how would one do multiple commands here?

I don't have a great suggestion for what to use instead. Perhaps something like this
```suggestion
    description: 'Prefix for the 'GAP' shell variable used by this action to launch GAP (e.g. setting this to "valgrind --trace-children=yes --leak-check=full" will run GAP through valgrind)'
```

---

A quote mark is used to escape a quote mark in literal strings in workflows and actions (see [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#literals). This seems very odd to me, so I'll try and find where this text actually gets rendered to double check.

As for your suggested change, I think you're right. I will add this in my revised PR.

Thank you for the review @fingolfin :)